app/helpers/interactive_page_helper.rb

```diff
@@ -1,13 +1,16 @@
 module InteractivePageHelper
   def runnable_activity_page_path(activity, page)
-    path_base = fetch_path_base(activity, @run, page)
-    pass_white_list_params(path_base) if path_base
+    path_base = fetch_path_base(@sequence, activity, @run, page)
+    append_white_list_params(path_base) if path_base
   end
 
-  def fetch_path_base(activity, run, page)  # move me to be private.
+  def fetch_path_base(sequence, activity, run, page)  # move me to be private.
     run = run_for_activity(activity, run)
```

Comment:
What is this run_for_activity doing here? We already have a run, so why do we need to find another one?

Reply:
This path might be for a different activity. So this method checks for a sequence_run and then finds the run for this sequence.  This is similar to the `runnable_activity_path` method, so perhaps it they could be combined better in the future.